editor/project_manager.cpp

```diff
@@ -494,6 +510,11 @@ class ProjectDialog : public ConfirmationDialog {
 
 			} else {
 				if (mode == MODE_NEW) {
+					if (create_dir->is_pressed()) {
+						Ref<DirAccess> d = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+						d->make_dir(project_path->get_text());
```

Comment:
So this is what I mentioned in my first comment. The folder shouldn't be created here, because the function may return later.